test/logging/README.md

```diff
@@ -0,0 +1,80 @@
+# Prerequisite
+
+- Kubernetes cluster is up and running
+- Install glusterfs-server in all nodes
+
+
+# Create glusterfs cluster
+
+
+- Create glusterfs cluster following instruction on [gluster-ansible](https://github.com/gluster/gluster-ansible) role
+- Create glusterfs endpoing and glusterfs service, refer [this](https://github.com/kubernetes/examples/tree/master/volumes/glusterfs) example
+
+
+# Create PV and PVC
+
+## Create PV
+```
+apiVersion: v1
```

Comment:
why cant add links to yaml files in here. why we need to maintain whole deployment sample

Reply:
Done